### PR TITLE
refactor: update jursdiction to civil (CMC-462)

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: compose/backend.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ services:
       DEFINITION_STORE_IDAM_KEY: "${IDAM_KEY_CCD_DEFINITION_STORE}"
       DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,xui_webapp
       USER_PROFILE_HOST: http://ccd-user-profile-api:4453
-      IDAM_USER_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_USER_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-web-public:8080}"
```
Comment (sabahirfan) [inline compose/backend.yml:37]: Is this required to change?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Unspecified civil money claims customised version of CCD Docker :whale:
+# Unspecified civil unspecified claims customised version of CCD Docker :whale:
```
Comment (ckalista) [inline README.md:1]: Take one `Undpecified` out :-)

File: compose/backend.yml (hunk 1 above)
Comment (dharmendrak) [reply compose/backend.yml:37]: yes, ccd have implemented open id connect

File: README.md (hunk 2 above)
Comment (dharmendrak) [reply README.md]: done

**Pull request merged**